cardano-cli/test/cardano-cli-golden/Test/Golden/CreateStaked.hs

```diff
@@ -0,0 +1,88 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Test.Golden.CreateStaked where
+
+import           Cardano.Api.Shelley (ShelleyGenesis (sgNetworkMagic, sgStaking))
+
+import           Cardano.Ledger.Crypto (StandardCrypto)
```

Comment:
`StandardCrypto` is exposed via `Cardano.Api.Ledger`. If you find yourself reaching for things directly from the ledger repo, check that it's not already exposed via `Cardano.Api.Ledger`.